src/librustc_session/config.rs

```diff
@@ -750,6 +751,8 @@ pub fn default_configuration(sess: &Session) -> CrateConfig {
         }
     }
 
+    ret.insert((Symbol::intern("panic"), Some(Symbol::intern(panic_strategy.desc()))));
```

Comment:
Please use `sym::panic` and modify `PanicStrategy` to also provide a pre-interned symbol (code is here: https://github.com/rust-lang/rust/blob/584e83dd5ae7a75b8214560c22eafbcfe153caa6/src/librustc_target/spec/mod.rs#L171-L176). Both `sym::abort` and `sym::unwind` are already available.